server/rest/src/main/java/org/infinispan/rest/resources/ServerResource.java

```diff
@@ -205,7 +205,8 @@ private CompletionStage<RestResponse> connectorStatus(RestRequest restRequest) {
 
       Json info = Json.object()
             .set("name", connectorName)
-            .set("ip-filter-rules", ipFilterRulesAsJson(connector));
+            .set("ip-filter-rules", ipFilterRulesAsJson(connector))
```

Comment:
usually this impacts a test too

Reply:
No, we don't have a test for this

Reply:
then we should add a test too. but IMO we need also the cache name and the cache list to get also the information without a server call. May I take in charge this part as a bigger thing than just having this information in this endpoint?